debian/changelog

```diff
@@ -1,3 +1,9 @@
+asterisk (8:19.6.0-1~wazo2) UNRELEASED; urgency=medium
```

Comment:
You should change the distribution from `UNRELEASED` to `wazo-dev-buster`. At the moment there's nothing in our build tools that would change it automatically.